routes/recipe.js

```diff
@@ -84,6 +102,7 @@ const getRecipeRouter = _ => {
 
     RecipeRouter.post('/', postRecipe);
     RecipeRouter.get('/:id', getRecipeByID);
+    RecipeRouter.get('/user/:id', getRecipesByUserID);
```

Comment:
I would change this to `.get('/user/:user_id', getRecipesByUserId);` then it makes cleaner above when you're using the param.

Reply:
Yes that makes sense. i will make the change. Thank you!